app/api/routers/event.py

```diff
@@ -23,7 +23,15 @@ async def handle_events(
     """
     events = premis_events.events
 
-    log.info(f"Got {len(events)} PREMIS-event(s).")
+    archived_events = [
+        event
+        for event in events
+        if event.is_valid and event.has_valid_outcome
+    ]
+
+    log.info(
+        f"Got {len(events)} PREMIS-event(s) of which {len(archived_events)} archived-events(s)."
```

Comment:
of which {len(archived_events)} archived-events(s). => of which {len(archived_events)} archived-events(s) with outcome OK.

As NOK events also get filtered :D